mobius-core/src/main/java/com/spotify/mobius/MessageDispatcher.java

```diff
@@ -51,12 +53,14 @@ public void accept(final M message) {
         new Runnable() {
           @Override
           public void run() {
-            try {
-              consumer.accept(message);
+            if (!disabled) {
```

Comment:
I think I would prefer to notify about the fact that a message arrived while disabled, at least at debug level. It's information that can be useful in some corner cases, so having it there may help troubleshooting.

Reply:
fixed